distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -240,4 +240,16 @@
 
     </fileSets>
 
+    <!-- We need this because the feature service is not up when we want to provision our
+    own config admin -->
+    <dependencySets>
```

Comment:
If we want to be more explicit about how the output file is named and what artifact is used, this is an alternative
```
    <!-- We need this because the feature service is not up when we want to provision our
 own config admin -->
    <dependencySets>
        <dependencySet>
            <outputDirectory>system/ddf/platform/osgi/platform-osgi-configadmin/${project.version}
            </outputDirectory>
            <outputFileNameMapping>
                ${artifact.artifactId}-${project.version}.${artifact.extension}
            </outputFileNameMapping>
            <includes>
                <include>ddf.platform.osgi:platform-osgi-configadmin:jar:${project.version}</include>
            </includes>
        </dependencySet>
    </dependencySets>
```

I would at least prefer the change to `include` since, right now, we aren't even specifying a version in our maven coordinates